components/tyk-pump/templates/deployment-pmp.yaml

```diff
@@ -145,6 +158,17 @@ spec:
           - name: TYK_PMP_PUMPS_MONGOAGGREGATE_META_USEMIXEDCOLLECTION
             value: "true"
           {{ end }}
+          {{ if has "mongo-selective" .Values.pump.backend }}
```

Comment:
We also need to enable selective pump when backend is set to `mongo`, right?

Reply:
updated in 5962d8348cbc956bde0f1c62067fedcafd2633ce, can you please check again?